src/detail-page/components/TabCategories.tsx

```diff
@@ -0,0 +1,19 @@
+type Category = {
+    name: string;
+    text: string;
+}
+
+export const categories: Category[] = [
+  {
+    name: 'learn',
+    text: 'What do I learn',
+  },
+  {
+    name: 'jobs',
+    text: 'Jobs',
+  },
+  {
+    name: 'resources',
+    text: 'Useful Resources',
+  },
+];
```

Comment:
Maybe this file content should be add to App state and passing in props?

Reply:
That's a very good point! However, we currently have quite a lot of changes going on in the app state (I mean PRs of Gosia and Agnieszka), so maybe we can leave it as is for now, to avoid more conflicts :) But we should probably review app state by the end of this sprint and decide how we want to refactor :) We may even consider if we want to add some state management tools ☝️

Reply:
Ok! So for now I move `categories` to props and remove `import { categories } from './tab-components/TabCategories';` in Tabs component. Then I the future we will add it to App state and pass it with the props.

Reply:
I move also file `TabCategories to data folder.